resources/nodes/Viewpoint.wrl

```diff
@@ -2,7 +2,7 @@
 
 Viewpoint {
   vrmlField SFFloat                                                                 fieldOfView            0.785398  # (rad)
-  vrmlField SFRotation                                                              orientation            0 0 1 0
+  vrmlField SFRotation                                                              orientation            -0.5773 0.5773 0.5773 2.0944
```

Comment:
That is not consistent with the documentation page.

Reply:
This odd rotation value doesn't look very nice. Is it to make it oriented exactly as previously? Can't we afford a nicer rotation here?

Reply:
> Is it to make it oriented exactly as previously?

No, I didn't like how it was before: x-east, y-up, and z-south (EUS). Now it is x-east, y-north, and z-up (ENU).

> Can't we afford a nicer rotation here?

No. If we want to keep a nice default orientation we have to use this one. It is not an implementation workaround, but ENU (world) - FLU (camera) workaround.

Reply:
As discussed, it might be better (although a bit arbitrary) to keep the 0 0 1 0 default orientation (as it is done in Gazebo and VRML97) to have the viewpoint from a human perspective rather than from a bird of prey perspective.

Reply:
And also this saves us from having long floating point values in the default orientation.

Reply:
But the nice default XY view from the top is also very convenient... So, let's keep it for now...

Reply:
In any case, we should improve the documentation about the Viewpoint, as it seems quite unclear right now.